config/v1/types_cluster_version.go

```diff
@@ -406,8 +420,19 @@ type ComponentOverride struct {
 type URL string
 
 // Update represents an administrator update request.
+// +kubebuilder:validation:XValidation:rule="has(self.architecture) && has(self.version) ? (size(self.architecture) == 0 || size(self.version) == 0) : true",message="cannot set both Architecture and Version"
```

Comment:
the godoc below talks about architecture and image not working properly together, but doesn't indicate that architecture and version cannot be set together.  Which is correct, the godoc or the CEL?

Reply:
updated godoc